pointercrate-core-pages/static/css/nav.css

```
@@ -202,8 +202,8 @@ footer .flex {
 
 footer .flex > * {
   margin: 1% 1%;
-  min-width: 150px;
-  max-width: 15%;
+  min-width: 200px;
+  max-width: 35%;
```

It's 3 things, so 3 * 35% > 100% (not sure if this would cause any problems, but lets be save)

```suggestion
  max-width: 30%;
```

---

like this?